github/github.go

```diff
@@ -154,6 +156,19 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// MediaRawType represents raw return of a request instead of json format
```

Comment:
Add a missing period at end of sentence and spell it as JSON, not json, since it's an acronym.